cf-agent/nfs.c

```diff
@@ -534,7 +535,7 @@ int VerifyNotInFstab(EvalContext *ctx, char *name, const Attributes *a, const Pr
 
     host = a->mount.mount_server;
     mountpt = name;
-
+    int changes = 0;
```

Comment:
Same here.

Reply:
Thanks for feedback, I will keep it in mind for the next time, but don't think it's big enough to do another amend + force push + re-review.